tests/pytests/test_json.py

```diff
@@ -201,6 +201,11 @@ def testSet(env):
     # Can also do multiple changes/side-effects, such as converting an object to a scalar
     env.execute_command('FT.CREATE', 'idx', 'ON', 'JSON', 'SCHEMA', '$.t', 'TEXT')
```

Comment:
Need to add some wildcard/union/slice/recursive-desent to a path in order to get multiple values
(for example, [testMultiPathResults](https://github.com/RedisJSON/RedisJSON/blob/8813a86e46765f3c411e5cc59bda704e9379fa60/tests/pytest/test.py#L859))